src/Examine.Core/Search/IFacetField.cs

```diff
@@ -0,0 +1,15 @@
+namespace Examine.Lucene.Search
+{
+    public interface IFacetField
+    {
+        /// <summary>
+        /// The field name
+        /// </summary>
+        string Field { get; }
+
+        /// <summary>
+        /// The field to get the facet field from
+        /// </summary>
+        string FacetField { get; set; }
```

Comment:
This is only ever set using the FacetField(string fieldName) methods. The interface doesn't need to expose this as get/set, it should only be get. The underlying implementation can make this get/set if it wants, or use a backing field.